test/lit.cfg

```diff
@@ -1155,7 +1165,10 @@ if run_vendor == 'apple':
         'Running tests on %s version %s (%s)' %
         (sw_vers_name, sw_vers_vers, sw_vers_build))
 
-    config.target_sdk_name = xcrun_sdk_name
+    if "-freestanding" in config.variant_suffix and config.swift_freestanding_is_darwin:
+        config.target_sdk_name = "freestanding"
```

Comment:
Can you add a comment here explaining why you are doing this.

Reply:
Added a comment in c843efb, also pulled the value from CMake in an attempt to show where that value comes from.